README.md

```diff
@@ -172,7 +172,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Codacy](https://www.codacy.com) [:rocket:](https://www.codacy.com/careers) | Automated code reviews & code analytics. | `Lisboa` `Remote` |
 | [Datadog](https://www.datadog.com) [:rocket:](https://www.datadoghq.com/jobs-europe) | Cloud monitoring as a service | `Remote` |
 | [SingleStore](https://www.singlestore.com/) [:rocket:](https://www.singlestore.com/careers/jobs/) | The scalable cloud database for data-intensive applications, deployed on AWS, Azure, or GCP. (formerly MemSQL) | `Lisboa` `Remote` |
-| [Oracle](https://www.oracle.com/index.html) [:rocket:](https://www.oracle.com/corporate/careers/) | DBMS's and information systems. | `Oeiras` `Remote` |
+| [Oracle](https://www.oracle.com/index.html) [:rocket:](https://www.oracle.com/corporate/careers/) | DBMS's and information systems. | `Lisboa` `Porto` `Remote` |
```

Comment:
```suggestion
| [Oracle](https://www.oracle.com/index.html) [:rocket:](https://www.oracle.com/corporate/careers/) | DBMS's and information systems. | `Oeiras` `Porto` <br> `Remote` |
```
This still seems Oeiras, any reason to change?

PS: Added <br> as defined in https://github.com/marmelo/tech-companies-in-portugal/blob/master/CONTRIBUTING.md

Reply:
Oeiras is in Lisbon (they call it Lisbon office on their sites). You are not making this distinction for other companies. For example: Capgemini engineering (former altran) is in Vila Nova de Gaia, but it is described as Porto.

Reply:
I am also adding the Porto (Maia) Office, which was not included on your list.

Reply:
It is still very far from Lisboa and Oeiras seems more appropriate.

Reply:
Whatever you say

![image](https://user-images.githubusercontent.com/13350752/197821496-eed13d3b-fd9e-4b30-b4be-c769813f9321.png)

Reply:
If the company calls it "Lisboa" that's fine by me.

PS: Still needs the <br>.